src/sourceTracking.ts

```diff
@@ -106,6 +107,54 @@ export class SourceTracking extends AsyncCreatable {
     this.subscribeSDREvents = options.subscribeSDREvents ?? false;
   }
 
+  /**
+   * A static method to help delete custom labels from a file, or the entire file if there are no more labels
+   *
+   * @param filename - a path to a custom labels file
+   * @param customLabels - an array of SourceComponents representing the custom labels to delete
+   */
+  public static async deleteCustomLabels(filename: string, customLabels: SourceComponent[]): Promise<void> {
```

Comment:
if it's not using anything from the class, it could be a standalone function, top-level exported if needed by the plugins.